packages/server/src/app/api/content/content.controller.ts

```diff
@@ -38,7 +38,7 @@ export class ContentController {
     }
 
     @Get('fetch-all-published')
-    async fetchAllPublished(@Request() req: any, @Query('pageNum') pageNum: number, @Query('userId') userId: string, @Query('kind') kind: ContentKind) {
+    async fetchAllPublished(@Request() req: any, @Query('pageNum') pageNum: number, @Query('userId') userId: string, @Query('kind') kind: ContentKind[]) {
```

Comment:
Array of `ContentKind`s now. Will make getting different combinations dead easy.